src/main/python/SystemML/mlcontext.py

```diff
@@ -20,20 +21,23 @@
 #-------------------------------------------------------------
 import os
 
-from py4j.java_gateway import JavaObject
+try:
+    from py4j.java_gateway import JavaObject
+except ImportError:
+    raise ImportError('Unable to import JavaObject from py4j.java_gateway. Hint: Make sure you are running with pyspark')
+    
 from pyspark import SparkContext
 import pyspark.mllib.common
-
+from pyspark.sql import DataFrame, SQLContext
+from .converters import *
 
 def dml(scriptString):
     """
     Create a dml script object based on a string.
-
```

Comment:
Nitpick: Can you add back these missing newlines from this file?  Just looks like an artifact from the merge.